src/Interpreter/interpreter.cpp

```diff
@@ -98,6 +98,12 @@ void Interpreter::setup_call(Module *m, uint32_t fidx) {
     // Push function locals
     for (uint32_t lidx = 0; lidx < func->local_count; lidx++) {
         m->sp += 1;
+#if DEBUG || TRACE || WARN || INFO
```

Comment:
I think we want this warning in production too, don't we? Anyway, I would change the message to "WebAssembly stack overflow ..." so it can't be confused with a vm bug.

Reply:
We could do it in production, but it might degrade performance slightly.

Reply:
So the rational is that we don't need error handling, because type checked code should not cause this error anyway?